prowler/providers/aws/services/documentdb/documentdb_instance_storage_encrypted/documentdb_instance_storage_encrypted.py

```diff
@@ -0,0 +1,29 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.documentdb.documentdb_client import (
+    documentdb_client,
+)
+
+
+class documentdb_instance_storage_encrypted(Check):
+    def execute(self):
+        findings = []
+        for db_instance in documentdb_client.db_instances:
+            report = Check_Report_AWS(self.metadata())
+            report.region = db_instance.region
+            report.resource_id = db_instance.id
+            report.resource_tags = db_instance.tags
+            report.resource_arn = documentdb_client.audited_account_arn
```

Comment:
Why are you setting here the AWS account ARN instead of the resource ARN?

Reply:
Due to the results being a global one, not based on each resource, check this comment I made in other one related with the same rule https://github.com/prowler-cloud/prowler/pull/2675#discussion_r1350463465